.textlintrc

```diff
@@ -0,0 +1,201 @@
+{
+  "filters": {
+    "node-types": {
+      "nodeTypes": ["BlockQuote"]
+    }
+  },
+  "rules": {
```

Comment:
quite opinionated though. it may turn out to be a royal pain in the @# to stick to those rules. also some of them are situation dependent, e.g. "validation done on the client side" and "client-side programming" are both correct I think. I'd never use hyphenation in the first sentence as in "validation done on the client-side". I think there's a grammar rule for that but I can't recall which---probably you hyphenate when the joined words are used as a compound adjective...
Anyhoo, I say let's give it a try for now and ditch it if it becomes a pain.

Reply:
i agree, i took them from jason, but lint:text is not part of the ci (you can use it locally if you wish)